test/test_torch.py

```diff
@@ -3889,6 +3889,44 @@ def test_comparison_ops_must_take_bool_output(self):
                        torch.logical_and, torch.logical_or, torch.logical_xor]:
                 self.assertEqual(op(torch.tensor([True]), torch.tensor([False])).dtype, torch.bool)
 
+        def test_out_comparison_ops_type_promotion_and_broadcasting(self):
+            # issue #42660
+            for op in [torch.lt, torch.le, torch.gt, torch.ge, torch.eq, torch.ne,
```

Comment:
What about extracting this list from the test and reusing it in the following tests?